app/api/custom/attendees.py

```diff
@@ -0,0 +1,45 @@
+from flask import Blueprint, request, jsonify, abort, make_response
+from flask_jwt_extended import current_user
+from sqlalchemy.orm.exc import NoResultFound
+
+from app.api.helpers.mail import send_email_to_attendees
+from app.api.helpers.permissions import jwt_required
+from app.api.helpers.errors import (
+    UnprocessableEntityError,
+    NotFoundError,
+    ForbiddenError,
+)
+from app.api.helpers.permission_manager import has_access
+
+from app.models.order import Order
+from app.models import db
+
+attendee_blueprint = Blueprint('attendee_blueprint', __name__, url_prefix='/v1')
+
+
+@attendee_blueprint.route('/attendees/send-receipt', methods=['POST'])
+@jwt_required
+def send_receipt():
+    """
+    Send receipts to attendees related to the provided order.
+    :return:
+    """
+    order_identifier = request.json.get('order-identifier')
+    if order_identifier:
+        try:
+            order = db.session.query(Order).filter_by(identifier=order_identifier).one()
+        except NoResultFound:
+            return NotFoundError({'parameter': '{order_identifier}'}, "Order not found").respond()
+
+        if (order.user_id != current_user.id) and (not has_access('is_registrar', event_id=order.event_id)):
+            return ForbiddenError({'source': ''},
+                                  'You need to be the event organizer or order buyer to send receipts.').respond()
+        elif order.status != 'completed':
+            abort(
+                make_response(jsonify(error="Cannot send receipt for an incomplete order"), 409)
+            )
+        else:
+            send_email_to_attendees(order, current_user.id)
+            return jsonify(message="receipt sent to attendees")
+    else:
+        return UnprocessableEntityError({'source': ''}, 'Order identifier missing').respond()
```

Comment:
Sure this is returning a proper error?

Reply:
@prateekj117 - Returning this

```
{
    "errors": [
        {
            "status": 422,
            "source": {
                "source": ""
            },
            "title": "Unprocessable Entity",
            "detail": "Order identifier missing"
        }
    ],
    "jsonapi": {
        "version": "1.0"
    }
}
```

Reply:
@codedsun If you have tested it, then ok. But make sure you tested it. Because UnprocessableEntityError is using jsonapi error internally. The same is with exceptions. That's why they don't work as expected in case of custom endpoints.

Reply:
So check other errors like `NotFoundError` as well. That's why they were not implemented in the old logic.

Reply:
@prateekj117 - Check 

![image](https://user-images.githubusercontent.com/20026440/72703921-050e5e80-3b7d-11ea-8fbd-a3e4f77fbac2.png)

Reply:
@prateekj117 - Not found error caase

![image](https://user-images.githubusercontent.com/20026440/72704244-363b5e80-3b7e-11ea-8fbd-f97b61421d3d.png)